shared/cluster.go

```diff
@@ -200,6 +202,23 @@ func FetchClusterIP(namespace string, serviceName string) (string, string, error
 	return ip, port, err
 }
 
+// FetchClusterIPs returns the cluster IPs and port of the service.
+func FetchClusterIPs(namespace string, svc string) (string, string, error) {
+	ip, err := RunCommandHost("kubectl get svc " + svc + " -n " + namespace +
+		" -o jsonpath='{.spec.clusterIPs[*]}' --kubeconfig=" + KubeConfigFile)
+	if err != nil {
+		return "", "", err
+	}
+
+	port, err := RunCommandHost("kubectl get svc " + svc + " -n " + namespace +
+		" -o jsonpath='{.spec.ports[0].port}' --kubeconfig=" + KubeConfigFile)
+	if err != nil {
+		return "", "", err
+	}
```

Comment:
Some example places where maybe using the `KubectlCommand` function would be good

Reply:
Actually I think I will move away from using `KubectlCommand` and write the cmd directly so that its easier to understand what command was run as part of the test

Reply:
and also printing out the command on failure would help with debugging

Reply:
That's fine with me -- we should standardize in that case across the repo (in a future PR)